src/concrete/ml/sklearn/base.py

```diff
@@ -1280,8 +1280,31 @@ def __init__(self, n_bits: int):
         #: The model's inference function. Is None if the model is not fitted.
         self._tree_inference: Optional[Callable] = None
 
+        #: Boolean that indicates whether the rounding feature is enabled or not.
+        self.__use_rounding: bool = True
+
         BaseEstimator.__init__(self)
 
+    def use_rounding(self, value: bool) -> None:
+        """Set the rounding feature.
+
+        Args:
+            value (bool): Whether to enable or disable rounding
+        """
+        assert isinstance(value, bool)
+
+        self.__use_rounding = value
```

Comment:
I think you've added too many underscores, only one _ is enough for private attributes !

Reply:
I don't agree.

This use_rounding must be hidden from the user.

Adding __ makes it more difficult to access the parameter.

Reply:
![image](https://github.com/zama-ai/concrete-ml/assets/14870263/f8be04a4-c101-44b9-a725-3d0c91169196)

Reply:
Got it